XenModel/Messages.resx

```diff
@@ -5754,7 +5754,7 @@ Would you like to eject these ISOs before continuing?</value>
     <value>E&amp;xpand All</value>
   </data>
   <data name="EXPIRED_CSS_UPSELLING_MESSAGE_HOST" xml:space="preserve">
-    <value>Your Customer Success Services (CSS) agreement has expired. This server will not receive any updates, including security updates</value>
+    <value>Your Customer Success Services (CSS) agreement has expired. This server will not receive any updates, including security updates.</value>

```

Comment:
I didn't add full stops on purpose since other sections in the tabs don't have them. Are you adding them because they're sentences?

Reply:
Yes, I added them because it's 2 sentences and it looked to me as if the full stop was forgotten for the second one.